Source: written by Niels Ole Salscheider
Upstream: yes, https://bugzilla.cyrusimap.org/attachment.cgi?id=1505
Reason: sasl.h uses size_t. Thus it has to include stddef.h

From 0eaadb2bad201140f2a6defbea33c20526d005ea Mon Sep 17 00:00:00 2001
From: Niels Ole Salscheider <niels_ole@salscheider-online.de>
Date: Sun, 25 Nov 2012 21:12:05 +0100
Subject: [PATCH] include/sasl.h: include stddef.h

Since b4744a25ebc11687def101ebbef8c82d3241d2a5, include/sasl.h uses
size_t. Thus, we have to include stddef.h.
---
 include/sasl.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/sasl.h b/include/sasl.h
index fef4d51..29bdab8 100755
--- a/include/sasl.h
+++ b/include/sasl.h
@@ -130,6 +130,7 @@
 #define SASL_VERSION_FULL ((SASL_VERSION_MAJOR << 16) |\
       (SASL_VERSION_MINOR << 8) | SASL_VERSION_STEP)
 
+#include <stddef.h>
 #include "prop.h"
 
 /*************
-- 
1.8.0

